docs/guides/admin/docs/releasenotes.md

```diff
@@ -4,15 +4,65 @@ Opencast 8: Release Notes
 Features
 --------
 
-- The `adminworker` distribution is no longer available.
+- Improved integration of external statistics providers (InfluxDB)
+  - Hourly scale for data exports
+  - Visualization of statistics in the Admin UI
```

Comment:
The Buildbot is complaining about more spaces on this indent: "Unordered list indentation [Expected: 4; Actual: 2]"

Reply:
So is Travis ;)